.github/ISSUE_TEMPLATE/release_tracker.yaml

```diff
@@ -0,0 +1,68 @@
+name: "Release Tracker Issue"
+description: "Track component releases and update manifests before the deadline"
+title: "[Release Tracker] - <Insert Release Name>"
+body:
+  - type: input
+    id: release_date
+    attributes:
+      label: "📅 Planned Release Date"
+      description: "Enter the planned release date (e.g., Mar 21st, 2025)"
+      placeholder: "Mar 21st, 2025"
```

Comment:
why need placeholder?

Reply:
They are grayed out fields which are overwritten when we enter a value, so I thought it will serve as an example